documentation/style-guide.rst

```diff
@@ -102,6 +102,47 @@ Unix
    The name of the operating system developed at AT&T Bell Labs in the early
    1970s.
 
+Diátaxis
+========
+
+Python's documentation strives to follow the :ref:`diataxis:diataxis`
```

Comment:
Here and below, let's not use intersphinx for the homepage, and just link directly. But there was a suggestion about linking elsewhere to https://diataxis.fr/tutorials/ via intersphinx, so that could use it. Otherwise we can remove it from `intersphinx_mapping`.

Reply:
Thanks @hugovk, I've changed that. I added an intersphinx link to tutorials at the end of that bullet point - apologies, must've missed that last time.